src/benchmarklib/encoding_config.cpp

```diff
@@ -4,7 +4,13 @@
 
 namespace hyrise {
 
-EncodingConfig::EncodingConfig() : EncodingConfig{SegmentEncodingSpec{EncodingType::Dictionary}} {}
+// Because it is hardly possible to save regular std:strings on disk, we use FixedStringDictionary as the default
```

Comment:
It's absolutely possible to persist them on disk, but doesn't help us here and needs some magic allocator stuff.

"hardly possible" is a strange wording. :)

(Not to be addressed)